tasks/install.yml

```diff
@@ -0,0 +1,38 @@
+---
+- block:
+  - name: Add apt key
+    apt_key:
+      url: http://packages.fluentbit.io/fluentbit.key
+      state: present
+
+  - name: Add repo
+    apt_repository:
+      repo: deb http://packages.fluentbit.io/ubuntu xenial main
+
+  - name: Install fluentbit
+    apt:
```

Comment:
Please add `update_cache` when installing package from recently added repository.

Reply:
update_cache has default value 'yes' in apt_repository module
[http://docs.ansible.com/ansible/latest/apt_repository_module.html](url)